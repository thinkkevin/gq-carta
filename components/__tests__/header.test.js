import React from "react";
import Header from "../header";
import Search from "../search";
import { shallow } from "enzyme";

describe("Test Header", () => {
  it("validates the header content using snapshot", () => {
    let comp = shallow(<Header />);
    expect(comp.html()).toMatchSnapshot();
    expect(comp.exists(".header")).toEqual(true); // check header class is set
    expect(comp.exists(Search)).toEqual(true);
  });
});
