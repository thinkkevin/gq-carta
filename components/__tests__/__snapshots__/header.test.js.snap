// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Header validates the header content using snapshot 1`] = `"<header class=\\"header\\"><a href=\\"/\\"><h1>Home</h1></a><div class=\\"search\\"><input type=\\"text\\" class=\\"input\\" name=\\"term\\" placeholder=\\"Search places\\" value=\\"coffee\\"/><span class=\\"submit\\">âš²</span><div class=\\"location\\">near <em></em></div></div></header>"`;
